src/Entity/ShippingSlotConfig.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' Shipping Slot plugin for Sylius.
+ *
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusShippingSlotPlugin\Entity;
+
+class ShippingSlotConfig implements ShippingSlotConfigInterface
+{
+    private ?int $id;
+    private ?string $name;
+    private ?array $rrules;
+    private ?int $preparationDelay;
+    private ?int $pickupDelay;
+    private ?int $durationRange;
+    private ?int $availableSpots;
+    private ?string $color;
```

Comment:
```suggestion
    private ?int $id = null;
    private ?string $name = null;
    private ?array $rrules = null;
    private ?int $preparationDelay = null;
    private ?int $pickupDelay = null;
    private ?int $durationRange = null;
    private ?int $availableSpots = null;
    private ?string $color = null;
```

Otherwise you'll have some issues, trust me ^^.